s2wav/transforms/analysis_jax.py

```diff
@@ -0,0 +1,106 @@
+from jax import jit, config
+
+config.update("jax_enable_x64", True)
+
+import jax.numpy as jnp
+from s2wav.utils import shapes
+from s2wav.filter_factory import filters
+import s2fft
+from functools import partial
+from typing import Tuple
+
+@partial(jit, static_argnums=(1, 2, 3, 4, 5, 6, 7, 8, 9, 10))
+def analysis_transform_jax(
+    f: jnp.ndarray,
+    L: int,
+    N: int = 1,
+    J_min: int = 0,
+    lam: float = 2.0,
+    spin: int = 0,
+    spin0: int = 0,
+    sampling: str = "mw",
+    nside: int = None,
+    reality: bool = False,
+    multiresolution: bool = False,
+    filters: Tuple[jnp.ndarray, jnp.ndarray] = None,
+) -> Tuple[jnp.ndarray, jnp.ndarray]:
+    r"""Wavelet analysis from pixel space to wavelet space for complex signals.
+
+    Args:
+        f (np.ndarray): Signal :math:`f` on the sphere with shape :math:`[n_{\theta}, n_{\phi}]`.
```

Comment:
Just a tiny thing - should change np.ndarray to jnp.ndarry in the docs

Reply:
Thanks!